org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizardPage.java

```diff
@@ -217,15 +239,54 @@ public void createControl(Composite parent) {
 		templateViewer = new ListViewer(list);
 		templateViewer.setContentProvider(new ArrayContentProvider());
 		templateViewer.setComparator(new ViewerComparator()); // default uses getLabel()/toString()
-		templateViewer.add("Loading templates");
-		templateViewer.getList().setEnabled(false);
 		templateViewer.addSelectionChangedListener(e -> {
 			setPageComplete(isPageComplete());
 		});
 		templateControlDecoration = new ControlDecoration(templateViewer.getControl(), SWT.TOP | SWT.LEFT);
 		templateControlDecoration.setImage(errorImage);
+		updateTemplateList();
+
+		//Update Template List with preferences change
+		IPreferenceStore store = AcutePlugin.getDefault().getPreferenceStore();
+		updateTemplatesListener = new IPropertyChangeListener() {
+			@Override
+			public void propertyChange(PropertyChangeEvent event) {
+				if (event.getProperty().equals(AcutePreferenceInitializer.explicitDotnetPathPreference)) {
+					updateTemplateList();
+				}
+			}
+		};
+		store.addPropertyChangeListener(updateTemplatesListener);
+
+		new Label(container, SWT.NONE);
+
+		Composite workingSetComposite = new Composite(container, SWT.NONE);
+		GridData layoutData = new GridData(SWT.FILL, SWT.FILL, true, false, 4, 1);
```

Comment:
Can you please add some verical spacing here, like 15px, to separate a bit working set from project creation as they are not so related.

Reply:
Sorry, somehow forgot this comment. Updating now.